service-commands/scripts/utils.sh

```diff
@@ -157,3 +162,53 @@ get_ip_addr() {
 format_bold() {
 	printf "$(tput bold)$@$(tput sgr0)"
 }
+
+configure_etc_hosts() {
+	read -p "Configure /etc/hosts? [y/N] " -n 1 -r && echo
+	if [[ "$REPLY" =~ ^[Yy]$ ]]; then
+		IP=$(get_ip_addr $provider $name)
+		echo "export AUDIUS_REMOTE_DEV_HOST=${IP}" >> ~/.zshenv
+		sudo node $PROTOCOL_DIR/service-commands/scripts/hosts.js remove
+		sudo -E AUDIUS_REMOTE_DEV_HOST=${IP} node $PROTOCOL_DIR/service-commands/scripts/hosts.js add-remote-host
+	fi
+}
+
+set_ssh_serveralive() {
+	if [[ ! -f "/etc/ssh/ssh_config.d/60-audius.conf" ]]; then
+		read -p "Configure /etc/ssh/ssh_config.d/60-audius.conf? [y/N] " -n 1 -r && echo
+		if [[ "$REPLY" =~ ^[Yy]$ ]]; then
+			echo "ServerAliveInterval 60" | sudo tee -a /etc/ssh/ssh_config.d/60-audius.conf
+		fi
+	fi
+}
+
+upload_gitconfig() {
+	if [[ -f "$HOME/.gitconfig.remote-dev" ]]; then
+		copy_file_to_remote $provider $user $name '~/.gitconfig.remote-dev' '~/.gitconfig'
+	fi
+}
```

Comment:
@cheran-senthil how do the above three functions look now?

They are all optional and to transfer `~/.gitconfig` requires manually creating `~/.gitconfig.remote-dev`.

Reply:
Yes, this is perfect 🔥